.gitignore

```diff
@@ -1 +1 @@
-.test
+test.md
```

Comment:
- [ ] Please add `node_modules` folder inside of .gitignore file, as you did with test.md